stdlib/email/src/main/ballerina/src/email/Module.md

```diff
@@ -0,0 +1,97 @@
+## Module overview
+
+This module contains functions to perform email operations such as sending and reading emails using SMTP, POP3, and IMAP4 protocols.
+
+## Sample
+
+The below sample uses the functions in the module to send an email using the SMTP protocol.
+
+```ballerina
+import ballerina/email;
+import ballerina/io;
+
+email:SmtpConfig smtpConfig = {
```

Comment:
Shall we use the most simple client (without this config) for the basic use case? We can specify the config details in the latter as more details. WDYT?

Reply:
Changed accordingly.